docs/running-headscale-container.md

```diff
@@ -101,7 +107,7 @@ tailscale up --login-server YOUR_HEADSCALE_URL
 To register a machine when running `headscale` in a container, take the headscale command and pass it to the container:
 
 ```shell
-docker exec headscale -- \
```

Comment:
Why do you remove these?

Reply:
Because (at least in docker 20.x) this doesn't work:
```bash
$ docker exec headscale -- headscale version
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "--": executable file not found in $PATH: unknown
$ docker exec headscale headscale version
dev
```

Reply:
those "--" also seem a bit fishy to me.

Reply:
Double dashes is a _really_ good convention in shell programs, [good elaboration here](https://www.baeldung.com/linux/double-dash-in-shell-commands).

But you are right, I was certain I had tested it with `docker` but it does not seem to work. Sad, why does docker not conform to standards.

Reply:
Really nice explanation, yeah docker is "special" in many different ways..